src/Input.js

```diff
@@ -8,6 +8,9 @@ class Input extends InputBase {
     if (this.props.type === 'static') { // eslint-disable-line react/prop-types
       deprecationWarning('Input type=static', 'StaticText');
       return <FormControls.Static {...this.props} />;
+    } else if (this.props.type === 'radio') {
+      deprecationWarning('Input type=radio', 'FormControls.RadioButton and FormControls.RadioGroup');
+      return <FormControls.RadioButton {...this.props} />;
```

Comment:
The docs are stull using this form so two things: 

1. maybe update the docs to not use Input?
2. this also needs to wrap the `RadioButton` in a `FormGroup` to be consistent with the old behavior